### DISCUSSION TITLE
Alert me when I am in a certain AWS profile

### DISCUSSION THREAD

**Discussion by skagedal** [starship/starship#5647] [Q&A] [answered]
Hello! I am looking for nice solutions to make myself very aware that the value of the environment variable `AWS_PROFILE` is set to `ScaryAdminProfile`. For example, turning the prompt red or changing the `>` to 'ADMIN >`, or something that stands out enough. 

How could I achieve something like that using starship?

**Comment by jankatins** [comment, marked as answer]
have a look at https://starship.rs/config/#environment-variable or https://starship.rs/config/#custom-commands. The former if just displaying something when an env var is set and the latter to basically compute everything which should be displayed in shell and then just let starship display it.

**Comment by skagedal** [comment]
Awesome, the custom commands is exactly what I need!
